src/Orleans.KafkaStreamProvider/KafkaQueue/TimedQueueCache/TimedQueueCache.cs

```diff
@@ -97,9 +97,9 @@ public class TimedQueueCache : IQueueCache
 
         public TimedQueueCache(QueueId queueId, TimeSpan cacheTimespan, int cacheSize, int numOfBuckets, Logger logger)
         {
-            _counterMessagesInCache = Metric.Context("KafkaStreamProvider").Counter($"Messages In Cache queueId:({queueId.GetNumericId()})", Unit.Items);
-            _meterCacheEvacuationsPerSecond = Metric.Context("KafkaStreamProvider").Meter($"Cache Evacuations Per Second queueId:({queueId.GetNumericId()})", Unit.Items);
-            _counterNumberOfCursorsCausingPressure = Metric.Context("KafkaStreamProvider").Counter($"Cursors causing pressure queueId:({queueId.GetNumericId()})", Unit.Items);
+            _counterMessagesInCache = Metrics.Metric.Context("KafkaStreamProvider").Counter($"Messages In Cache queueId:({queueId.GetNumericId()})", Unit.Items);
```

Comment:
You can change the code to use single metrics context.

var context = Metrics.Metric.Context("KafkaStreamProvider");
context.Counter(......